idpasslite/install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+#
+
+# Manually install a jar file into ~/.m2/ local repository
+# and then reference it in pom.xml by:
+#
+#   <dependency>
+#       <groupId>newlogic.com</groupId>
+#       <artifactId>idpass-lite-mosip</artifactId>
+#       <version>1.0.0</version>
+#   </dependency>
+#
+#   <dependency>
+#       <groupId>com.google.protobuf</groupId>
+#       <artifactId>protobuf-java</artifactId>
+#       <version>3.12.2</version>
+#   </dependency>
+
+mvn install:install-file \
```

Comment:
`idpass-lite.jar` to be hosted online and added into `pom.xml` as dependency. Will change its `groupId` , `artifactId`